mubench.reviewsite/init_db.sql

```diff
@@ -83,6 +83,23 @@ CREATE TABLE IF NOT EXISTS `types` (
   UNIQUE KEY `name` (`name`)
 ) ENGINE=MyISAM  DEFAULT CHARSET=latin1;
 
+CREATE TABLE IF NOT EXISTS `misuse_tags` (
+  `exp` varchar(10) NOT NULL,
+  `detector` varchar(100) NOT NULL,
+  `project` varchar(100) NOT NULL,
+  `version` varchar(100) NOT NULL,
+  `misuse` varchar(100) NOT NULL,
+  `tag` int(11) NOT NULL,
+  KEY `lookup` (`exp`,`detector`,`project`,`version`,`misuse`)
```

Comment:
You could include `tag` in the key and make this a PRIMARY KEY, since we can assign each tag only once. This will (for maaaaaany tags ;)) improve the efficiency of the delete query and prevent duplicated tags. Since this is a N:M linking table, its the usual procedure to have the key span the entire table (unless the link itself has additional properties).